docs/content/docs/connectors/flink-sources/postgres-cdc.md

```diff
@@ -236,6 +236,13 @@ Connector Options
           so it does not need to be explicitly configured 'execution.checkpointing.checkpoints-after-tasks-finish.enabled' = 'true'
       </td>
     </tr>
+    <tr>
+      <td>checkpoint.cycle</td>
```

Comment:
Naming is hard, how about `scan.lsn-commit.checkpoints-num-delay` ?

Reply:
Thanks @leonardBang, it is clearer and describes the intent. I'll refactor it accordingly